sbe/__init__.py

```diff
@@ -1004,6 +1024,8 @@ def _parse_schema(f: TextIO) -> Schema:
                     x.semanticType = attrs['semanticType']
                 if 'presence' in attrs:
                     x.presence = PRESENCE_TYPES[attrs['presence']]
+                if 'offset' in attrs:
+                    x.padding = int(attrs['offset']) - stack[-1].size()
```

Comment:
What if `stack[-1]` is the parent element? Can that happen?

Reply:
That's actually the intent. MDP3 has a case like this:
```
<composite name="groupSize8Byte" ...>
  <type name="blockLength" ... primitiveType="uint16">
  <type name="numInGroup" ... offset="7" primitiveType="uint8">
</compossite>
```
When we get to `numInGroup`, we need to know that the parent element size is currently 2 (occupied by the `blockLength` element), and we need to expect 5 bytes of padding before the `numInGroup` element which located 7-bytes from the start of `groupSize8Byte` (the parent).